xline/src/storage/kv_store.rs

```diff
@@ -960,15 +994,23 @@ mod test {
                 }
                 .into(),
             );
-            let _cmd_res = store.execute(&req)?;
-            let (_sync_res, ops) = store.after_sync(&req).await?;
-            store.db.flush_ops(ops)?;
+            sync_req(&store, &req).await?;
         }
         Ok(store)
     }
 
+    async fn sync_req(
```

Comment:
the inner operations are not "syncing" but, execute "after_sync" step and flush to persistent store. Shall we rename it "exe_as_and_flush"?

Reply:
Renamed